cli/azd/cmd/util.go

```diff
@@ -449,9 +449,10 @@ func promptTemplate(ctx context.Context, message string, askOne Asker) (string,
 	var selectedTemplateIndex int
 
 	if err := askOne(&survey.Select{
-		Message: message,
-		Options: templateNames,
-		Default: templateNames[0],
+		Message:    message,
+		Options:    templateNames,
+		Default:    templateNames[0],
+		FooterText: withHighLightFormat("... scroll down to see more options below ..."),
```

Comment:
This would be the only required change if the feature is implemented/accepted from AlecAivazis/survey lib